### DISCUSSION TITLE
Downloading a raw file does not apply .gitattributes text eol

### DISCUSSION THREAD

**Discussion by francisdb** [community/community#76759] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When clicking on the "Download raw file" button the download does not have the .gitattributes rules applied.

For example we have this in `.gitattributes`:

```
# Set the default behavior, in case people don't have core.autocrlf set.
*               text=auto

*.vbs		text eol=crlf
*.vbs.original  text eol=crlf

# do not touch patch files as they are mixed LF/CRLF
*.patch         -text
```

All `.vbs` files have been normalized and a git clone gives us `CRLF` line endings. However if we download the file from the github website the file has `LF` line endings.

**Comment by ghost** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> When clicking on the "Download raw file" button the download does not have the .gitattributes rules applied.
> 
> For example we have this in `.gitattributes`:
> 
> ```
> # Set the default behavior, in case people don't have core.autocrlf set.
> *               text=auto
> 
> *.vbs		text eol=crlf
> *.vbs.original  text eol=crlf
> 
> # do not touch patch files as they are mixed LF/CRLF
> *.patch         -text
> ```
> 
> All `.vbs` files have been normalized and a git clone gives us `CRLF` line endings. However if we download the file from the github website the file has `LF` line endings.

**Comment by francisdb** [comment]
Looks like uploading a file has the same issues
https://github.com/jsm174/vpx-standalone-scripts/pull/183#issuecomment-1845430492
